.overcommit.yml

```diff
@@ -33,19 +33,6 @@ PreCommit:
     include: '**/*.slim'
 
 CommitMsg:
-  ALL:
-    on_warn: 'fail'
-
```

Comment:
This is the only part that needs to be reverted. We still want the MessageFormat below to show up as a warning. We just don't want it to fail.